Android/Fundamentals/intro-to-android/android-versions.md

```diff
@@ -0,0 +1,49 @@
+# android-versions
+author: tlawson
+
+levels:
+
+  - beginner
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[Different Versions of Android](https://developer.android.com/about/dashboards/index.html)'
+
+---
+## Content
+
+Each Android has been given a recognizable name which falls in alphabetical order named after some type of dessert. There have been over 12 different versions of Android, Oreo being the latest publicly available one.
+
+Android first started out with Cupcake and Donut, but the first release of Android was Froyo which was version 2.2. Froyo provided huge speed, memory and performance, push notifications and switching between multiple keyboards and languages. Google released version 2.3 which was Gingerbread. This release effort was for user interface and included an implementation to allow field communication, larger screen resolution, and support for multiple cameras. Honeycomb version 3.0 was created which was specific to tablet devices including elements of the new holographics.
```

Comment:
On a phone, we want to reduce the walls of text as much as possible, and strive for relevance. The most recent versions are the most relevant, so let's list out only the major versions that are still in high circulation.

Also I think this would be best as a bulleted list ( I did the first few because they were simple)

Each successive android version has added features and support for more devices. 

Cupcake - Not released
Doughnut - Not released
Froyo
- memory and performance improvements
- push notifications
- multiple keyboards
- internationalization
Gingerbread
- Near Field Communication API
- Large screen resolution support
- Multi-camera support

etc


Differentiating between current versions that are out is really valuable, but because this article will have to be kept up to date, let's just keep it to the last 2 or 3 versions. (or however many are significant market shares)